src/view/components/toolbar/motion/Accelerometer.tsx

```diff
@@ -51,56 +54,77 @@ interface IProps {
 }
 
 const GESTURE_BUTTON_MESSAGE = "Send Gesture";
+const MANUAL_ACCELERATION_MESSAGE = "Set the acceleration manually here";
 
-export const Accelerometer: React.FC<IProps> = (props: IProps) => {
-    return (
-        <div className="AccelerometerBar">
-            <br />
-            {/* Implement Gestures Here */}
-            <Dropdown options={GESTURES} onSelect={props.onSelectGestures} />
-            <SensorButton
-                label={GESTURE_BUTTON_MESSAGE}
-                onMouseDown={() => {
-                    if (props.onSendGesture) {
-                        props.onSendGesture(true);
-                    }
-                }}
-                onMouseUp={() => {
-                    if (props.onSendGesture) {
-                        props.onSendGesture(false);
-                    }
-                }}
-                onKeyDown={(e: React.KeyboardEvent) => {
-                    handleOnKeyDown(e, props.onSendGesture);
-                }}
-                onKeyUp={(e: React.KeyboardEvent) => {
-                    handleOnKeyUp(e, props.onSendGesture);
-                }}
-                type="gesture"
-            />
-            <ThreeDimensionSlider
-                axisProperties={MOTION_SENSOR_PROPERTIES}
-                onUpdateValue={props.onUpdateValue}
-                axisValues={props.axisValues}
-            />
-        </div>
-    );
-};
-const handleOnKeyDown = (
-    e: React.KeyboardEvent,
-    onSendGesture?: (isActive: boolean) => void
-) => {
-    if (e.key === CONSTANTS.KEYBOARD_KEYS.ENTER) {
-        console.log("gestures");
-        if (onSendGesture) onSendGesture(true);
-    }
-};
-const handleOnKeyUp = (
-    e: React.KeyboardEvent,
-    onSendGesture?: (isActive: boolean) => void
-) => {
-    if (e.key === CONSTANTS.KEYBOARD_KEYS.ENTER) {
-        console.log("gesturesUp");
-        if (onSendGesture) onSendGesture(false);
+export class Accelerometer extends React.Component<IProps> {

```

Comment:
Do we have any tests for the sensor modals?

Reply:
Added testing for the accelerometer/gestures :)